src/components/admin/hazardous-materials-specifiers.tsx

```diff
@@ -8,23 +8,26 @@ import { useAppSelector } from "app/hooks";
 import { MaterialItem, MaterialText } from "../../styled/layout-components/material-item";
 import { selectKeycloak } from "features/auth-slice";
 import { ErrorContext } from "components/error-handler/error-handler";
-import { WasteSpecifier } from "generated/client";
+import { LocalizedValue, WasteSpecifier } from "generated/client";
+import { selectLocaleState } from "features/locale-slice";
+import LocalizationUtils from "utils/localization-utils";
 
 /**
  * Component for hazardous waste specifier dropdown menu editor
  */
 const HazardousMaterialsSpecifiers: React.FC = () => {
   const errorContext = React.useContext(ErrorContext);
   const keycloak = useAppSelector(selectKeycloak);
-
+  const selectedLanguage = useAppSelector(selectLocaleState).language;
```

Comment:
It would be better to use direct selector, or in other words something like
`useAppSelector(selectSelectedLanguage)`;